src/app/router/connectorRoutes.tsx

```diff
@@ -93,6 +110,7 @@ function getConnectorRoutes() {
       ),
       logo: lndhubGo,
     },
+    ...(isDevelopment ? kollider : []),
```

Comment:
Is this a left over or should it stay?

Reply:
As mentioned above, the Kollider connector is not full integrated yet.
Therefore, it is hidden on production.

Thanks for testing the makePayment functionality.
I added it to the open todos.